exercises/practice/change/.approaches/introduction.md

```diff
@@ -0,0 +1,27 @@
+# Introduction  
+
+There is an idiomatic approach to solving "Change."  
+You can use [dynamic programming][dynamic-programming] to calculate the minimum number of coins required for a given total.
+
+## General guidance
+
+The key to solving "Change" is understanding that not all totals can be reached with the available coin denominations.
+The solution needs to figure out which totals can be achieved and how to combine the coins optimally.
+
+## Approach: Dynamic Programming
+
```

Comment:
Looking at the other approaches, this section should just show the code for the dynamic programming approach, followed by a the like to the dynamic programming content.

Reply:
I've updated it to include only the code after the heading, followed by a single sentence with a link to the approach.